src/store/UserSlice.ts

```diff
@@ -14,6 +15,7 @@ const initialState = {
   username: '',
   name: '',
   email: '',
+  host: false,
```

Comment:
did this fix the host thing?

Reply:
no, it did not. i decided to scrape the host thing for now